spec/protocol-spec/main.conf

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This is a temporary config. We do not have to merge it.

Reply:
ok